jinahub/indexers/searcher/compound/FaissPostgresIndexer/README.md

```diff
@@ -61,14 +61,24 @@ with get_my_flow() as flow:
     flow.post(
         on='/sync',
         # we need some specific parameters here
+        # startup is only needed the 1st time
```

Comment:
Could you explain in one sentence why? Not sure if it helps the user, but it might.

Reply:
Re-check